inst/include/conditionalSampler.h

```diff
@@ -0,0 +1,673 @@
+// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; indent-tabs-mode: nil; -*-
+//
+// sampler.h: Rcpp integration of SMC library -- sampler object
+//
+// Copyright (C) 2021 Adam Johansen, Dirk Eddelbuettel, Leah South, Ilya Zarubin
+//
+// This file is part of RcppSMC.
+//
+// RcppSMC is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// RcppSMC is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with RcppSMC. If not, see <http://www.gnu.org/licenses/>.
+
+//! \file
+//! \brief Defines the overall sampler object.
```

Comment:
It would be good for the one-line summary of what the file does to be different to that for sampler.h

Reply:
Done!